src/amo/reducers/landing.js

```diff
@@ -0,0 +1,36 @@
+import {
+  LANDING_GET,
+  LANDING_LOADED,
+  LANDING_FAILED,
+} from 'core/constants';
+
+
+export const initialState = {
```

Comment:
We use `entities` in the `LOADED` reducer to create our `results` object, yeah. But we don't use it/access it outside the reducer/in the components.